modules/jvm/api/tests/features/performance.feature

```diff
@@ -0,0 +1,14 @@
+# builder only
+@ubi8/openjdk-11
+@ubi8/openjdk-17
+@ubi8/openjdk-21
+Feature: JVM Performance tests
```

Comment:
This isn't a performance test. it was originally added as a regression test for https://issues.redhat.com/browse/CLOUD-2070, and relates to permissions issues for a running user.
The test also is broken at the moment for unknown reasons. the code it was testing was related to changing permissions on `/etc/passwd` and has been removed. I think we should just delete this test please.